db/model.js

```diff
@@ -1,9 +1,21 @@
 const db = require('./connection.js')
+const fs = require('fs').promises;
 
 const selectAllTopics = () => {
     return db.query(`SELECT * FROM topics;`).then((result) => {
         return result.rows;
     })
 }
 
-module.exports = selectAllTopics;
\ No newline at end of file
+const selectAllEndpoints = async () => {
```

Comment:
As we are not interacting with the database, this should be handled in the controller